.github/workflows/prepare_release.yaml

```diff
@@ -0,0 +1,80 @@
+name: Prepare release
```

Comment:
this workflow is used to update the version and build the apidocs on a manual trigger (by giving the new version as well). It then opens a PR on main